doc(release): prepare release notes for MAP 22.04.4 (#2076)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md

## Centreon MAP

### 22.04.4

Release date : `soon`

#### Bug fixes

> - Fixed an issue preventing users to create a map with the API

Reviewer: ```suggestion
- Fixed an issue preventing users to create a map with the API.
```

## Centreon MAP

### 22.04.4

Release date : `January 20, 2023`

#### Bug fixes

- Fixed an issue preventing users to create a map with the API.
- Fixed an issue that caused server to fail to start when trying to load empty output.
- Fixed links where empty bendpoints caused maps to become not editable.
- Resource name properties are now used for links when saving maps.
- Fixed an issue that caused metric links in the same views to display the same value after the first refresh.
- Fixed server startup error due to duplicate key.

> - Fixed an issue that could cause layers to not be applied properly.

Reviewer: ```suggestion
- Fixed an issue that could prevent the correct application of layers.
```

> - Fixed an issue that caused centreon-map-engine to require java-17-openjdk-devel rpm and prevented successful installation.

Reviewer: ```suggestion
- Fixed an issue that caused MAP Engine server to require java-17-openjdk-devel RPM and prevented successful installation.
```

> - Fixed an issue that cause modifications to resource names in editor to not be properly updated after re opening editor.

Reviewer: ```suggestion
- Fixed an issue where resource names modifications in the editor were not correctly updated after reopening the editor.
```
